src/components/organisms/Table/v2/Badge.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// T Y P E S
+
+export interface BadgeProps {
+  value: string | boolean | number
```

Comment:
Why not put this component in the `atom` folder?

Reply:
yeah, thanks for the reminder